### DISCUSSION TITLE
Creating line-items with multiple keys/values

### DISCUSSION THREAD

**Discussion by umutkirgoz** [prebid/line-item-manager#64] [General]
Hi;

I want to create separate order sets for different bidders with the targeting key/values like seen below. Is it possible to create these line items by using line-item-manager. I cannot out this. Any suggestions?

**Line Item Targeting for Bidder_1 (bidder_1)**
hb_bidder=bidder_1
hb_pb=0.01

**Line Item Targeting for Bidder_2 (bidder_2)**
hb_bidder=bidder_2
hb_pb=1.10

I donot want to use single targeting key like **hb_pb_bidder_1**. I want to use separate global keys for bidder name **(hb_bidder)** and price bucket **(hb_pb)**

Regards...

**Comment by dshore** [comment]
I may not understand your question completely but I believe you just want to use the "single order" command line option:
```
-s, --single-order           Create a single set of orders instead of orders per bidder.  [default: False]
```
This creates targeting using 'hb_pb' and 'hb_bidder' will equal the bidder code as part of Prebid ad server targeting.
